client/README.md

```diff
@@ -53,3 +53,23 @@ For example to access the `staging` server (the default), the files are:
 Android: client/android/app/src/<flavor>/google-services.json
 iOS:     client/ios/config/<flavor>/GoogleService-Info.plist
 ```
+
+##### Using the Firebase Emulators
+
+If you'd like to test your app without using a "real" Firebase project, you can use the Firebase Local Emulator Suite as follows.
+
+First, start your local emulators by navigating to your `server/functions` directory and running:
+
+    firebase emulators:start --project=dev
+
+Then, in your `client` directory, update your `main.dart` as follows:
+
+    const USE_FIREBASE_LOCAL_EMULATORS = true;
+
+Then, run your application in its `hack` flavor:
+
+    flutter run --flavor=hack
```

Comment:
Why the `hack` flavor? That's really meant for penetration testing. I was expecting people to use the staging flavor.... which I think it should default to anyway.

Reply:
Currently the only flavor using our `v2` backend is `hack` (thanks to Bez's #1971), but once the other endpoints are removed in favor of direct Firestore use we should be able to replace this with any flavor we want.